README.md

```diff
@@ -2435,21 +2435,19 @@ type Strings = ElementType<string[]>; // string
 
 In TypeScript, predefined conditional types are built-in conditional types provided by the language. They are designed to perform common type transformations based on the characteristics of a given type.
 
-Exclude<UnionType, ExcludedType>: This type removes all the types from Type that are assignable to ExcludedType.
+`Exclude<UnionType, ExcludedType>`: This type removes all the types from Type that are assignable to ExcludedType.
 
-Extract<Type, Union>: This type extracts all the types from Union that are assignable to Type.
```

Comment:
Did you intend remove thus point? May I ask why?

Reply:
It wasn't me intention :)